src/entry-points/popup/App.svelte

```diff
@@ -400,6 +402,22 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
   $: controllerTypeAlwaysSounded = latestTelemetryRecord?.controllerType === ControllerKind_ALWAYS_SOUNDED;
 
   const displayNewBadgeOnExperimentalAlgorithm = new Date() < new Date('2024-09-30');
+
+  function onAdvancedModeChange() {
+    (createOnInputListener('advancedMode'))();
+    if (this.checked) {
+      settings.experimentalControllerType = ControllerKind_STRETCHING;
+      settings.marginBefore = 0;
+    }
+  }
+
+  function onSimpleSliderInput() {
+    (createOnInputListener('simpleSlider'))();
```

Comment:
Is there a way to make it less ugly? Creating a listener and immediately invoking it. Not great.